identity/did/factory_service.go

```diff
@@ -2,68 +2,181 @@ package did
 
 import (
```

Comment:
I dont think the name of this file should be `factory_service`, instead service as we talked about?